cypress/integration/seededFlows/policyFlows/limitPostCreationToAdmins.spec.js

```diff
@@ -17,7 +17,7 @@ describe('Limit Post Creation to Admins', () => {
     // "Create Button" that's on the nav bar but not in the drop down.
 
     cy.findByRole('button', { name: 'Navigation menu' }).as('menuButton');
-  cy.get('@menuButton')
+    cy.get('@menuButton')
```

Comment:
I'm guessing this little change popped in as an autofix when I merged main into my branch 🤷‍♀️